examples/kms/vault/csi-kms-connection-details.yaml

```diff
@@ -74,5 +74,14 @@ data:
       "encryptionKMSType": "aws-sts-metadata",
       "secretName": "ceph-csi-aws-credentials"
     }
+  kmip-test: |-
+    {
+      "KMS_PROVIDER": "kmip",
+      "KMIP_ENDPOINT": "kmip:5696",
+      "KMIP_SECRET_NAME": "ceph-csi-kmip-credentials",
```

Comment:
**Just fyi**, since we will be creating this Secret from UI, so will add some random generated text to the Secret's name and pass that name to this "KMIP_SECRET_NAME" key.
Reason for adding random text to Secret's name is that, user can create multiple KMS connections, if 2 different connections will have same name for Secret it could be an issue.

Reply:
also, Secret will be in deployment namespace, `openshift-storage` in our case...

Reply:
Yes, and yes
Both sounds good to me.